### PR TITLE
Hide dotfiles and/or gitignored, changelog, and a bugfix

### PR DESCRIPTION
**Pull request by sum01**
Unsure if you just want me to merge, tag, and push without review, so I'm posting a PR for this.

Adds two new options: `filemanager-showdotfiles` and `filemanager-showignored` (true by default)
Lets you hide dotfiles and/or vcs-ignored (only git at the moment). You can mix-and-match the options for your desired output.

### CODE REVIEW

File: filemanager.lua [hunk 1]
```diff
@@ -62,43 +72,143 @@ local function is_dir(path)
...
+	-- TODO: Support more than just Git, such as Mercurial or SVN
+	if has_git() then
+		-- If the dir is a git dir, get all ignored in the dir
+		local git_ls_results =
+			get_popen_readout('git -C "' .. tar_dir .. '" ls-files . --ignored --exclude-standard --others --directory')
+		-- Cut off the newline that is at the end of each result
+		for split_results in string.gmatch(git_ls_results, "([^\r\n]+)") do
+			-- git ls-files adds a trailing slash if it's a dir, so we remove it (if it is one)
+			readout_results[#readout_results + 1] =
+				(is_dir(JoinPaths(tar_dir, split_results)) and string.sub(split_results, 1, -2) or split_results)
```
Comment (sum01) [inline filemanager.lua]: Hmm. Actually I should just check for a trailing slash instead of an `is_dir` call.

**Comment by sum01** [comment]
This should actually be done now, unless I find another bug to fix or something 😄 

Note that with the CHANGELOG you'll have to do a few things before tagging. [This is what I normally do when tagging](https://github.com/sum01/fmt-micro/commit/de794262d399cf130556b1ed0f1a775ee2bf1dc7)

**Comment by sum01** [comment]
@NicolaiSoeborg @tommyshem 
This PR brings 2 bugfixes (along with enhancements). Could I get a yes/no if I should merge/push/tag without review? If it's ok, I'll do it myself.

**Comment by tommyshem** [comment]
@sum01 Looks OK to me but its not my repo to give permission.
@NicolaiSoeborg sum01 needs permission to push or review by you.

**Pull request merged**